actgen/train.py

```diff
@@ -106,12 +44,11 @@ def setup(self):
 
         if self.params['oracle'] and not self.params['dqn_train_pin_other_q_values']:
             raise RuntimeError('dqn_train_pin_other_q_values must be set to true when performing oracle action generalization')
-        
-        self.params['device'] = utils.determine_device(self.params['disable_gpu'])
-        logging.info('training on device {}'.format(self.params['device']))
+
+        self.determine_device()
 
         if self.params['agent'] == 'random':
-            self.agent = RandomAgent(env.observation_space, env.action_space)
+	        self.agent = RandomAgent(env.observation_space, env.action_space)
```

Comment:
Did an extra space get added here by mistake?

Reply:
good catch, not sure how that got changed